SunEngine.Core/DataBase/DataBaseConnection.cs

```diff
@@ -88,7 +101,15 @@ public void UpdateSequence(string tableName, string keyName)
 
         public bool IsPostgres()
         {
-            return DataProvider.Name.StartsWith("Postgre");
+            return NameSqlProvider == TypeSqlProvider.PostgreSql;
         }
     }
-}
+    
+    [Flags]
+    public enum TypeSqlProvider
+    {
+        MySql = 1,
+        PostgreSql = 2,
```

Comment:
You can don`t specify a value for enum items after first element(need set a start value, if it not 0).

Reply:
@kreuwont it's good practice to specify, it would help in case of future changes, making old code compatible,